docs/installation.md

```diff
@@ -29,6 +29,9 @@ npm install -g @asyncapi/cli
 ## Mac
 There are two ways to install the AsyncAPI CLI on your macOS: using the `brew` package manager or `pkg` files.
 
+### Windows 
```

Comment:
wrong location of section, Windows should not be under Mac

also please provide direct links to 64 and 32 bit files like we do for other distros -> https://github.com/asyncapi/cli/releases/latest/download/asyncapi.x64.pkg

these can be "predicted"